dev/com.ibm.ws.security.oauth/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -344,6 +344,9 @@ internalClientId.desc=The ID of an existing client that will be used to generate
 internalClientSecret=Client secret used to generate tokens
 internalClientSecret.desc=The secret of an existing client that will be used to generate tokens for PersonalTokenManagement and UsersTokenManagement UI pages.
 
+ropcPreferUserSecurityName=Resource Owner Password Credentials Grant (ROPC) Prefers User Security Name
+ropcPreferUserSecurityName.desc=Boolean indicating that the user name in tokens created by the ROPC Grant type will be set to the User Security Name if it differs from the user name supplied in the ROPC request. The default is false.
```

Comment:
How does this sound for the description? Is it still technically correct? 
"If the user security name differs from the username that is supplied in the ROPC request, then the username is set to the user security name for all tokens that are created by the ROPC grant type. The default is false."

Reply:
That'll work.  Done.